src/sass/_Fabric.Typography.Mixins.scss

```diff
@@ -5,7 +5,7 @@
 // --------------------------------------------------
 // Fabric Core Typography mixins
 
-@mixin ms-baseFont() {
+@mixin ms-baseFont {
   font-family: $ms-font-family-base;
```

Comment:
I'll take it out for now then and we can discuss more. I definitely want to set the base font family here, as it will greatly simplify how we do localization and even (finally!) allow for localized fonts in the Fabric JS components.